Gifski/ConversionViewController.swift

```diff
@@ -77,24 +79,39 @@ final class ConversionViewController: NSViewController {
 		timeRemainingEstimator.start()
 
 		progress?.performAsCurrent(withPendingUnitCount: 1) { [weak self] in
-			Gifski.run(conversion) { result in
-				guard let self = self else {
-					return
-				}
-
+			guard let self = self else {
+				return
+			}
+			let gifUrl = self.generateTempGifUrl(for: conversion.video)
+			if let gifData = gifData {
 				do {
-					let gifUrl = self.generateTempGifUrl(for: conversion.video)
-					try result.get().write(to: gifUrl, options: .atomic)
+					try gifData.write(to: gifUrl, options: .atomic)
+
 					try? gifUrl.setMetadata(key: .itemCreator, value: "\(App.name) \(App.version)")
 					defaults[.successfulConversionsCount] += 1
 
 					self.didComplete(conversion: conversion, gifUrl: gifUrl)
 				} catch Gifski.Error.cancelled {
-					self.cancelConversion()
+					cancelConversion()
```

Comment:
I think this case shouldn't happen for `gifData` - `try gifData.write` shouldn't throw `Gifski.Error.cancelled` so we may remove this specialized `catch`